src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamTestRunner.java

```diff
@@ -79,32 +79,32 @@
 /*
  * @author msgroi
  */
-class MTAmazonDynamoDBStreamTestRunner {
+class MtAmazonDynamoDbStreamTestRunner {
 
-    private static final Logger log = LoggerFactory.getLogger(MTAmazonDynamoDBTestRunner.class);
-    private final AmazonDynamoDB mtAmazonDynamoDB;
+    private static final Logger log = LoggerFactory.getLogger(MtAmazonDynamoDbTestRunner.class);
+    private final AmazonDynamoDB mtAmazonDynamoDb;
     private StreamWorker streamWorker;
-    private List<MTRecord> expectedMTRecords;
+    private List<MtRecord> expectedMtRecords;
 
-    MTAmazonDynamoDBStreamTestRunner(AmazonDynamoDB mtAmazonDynamoDB,
-                                     AmazonDynamoDB rootAmazonDynamoDB,
-                                     AmazonDynamoDBStreams rootAmazonDynamoDBStreams,
+    MtAmazonDynamoDbStreamTestRunner(AmazonDynamoDB mtAmazonDynamoDb,
+                                     AmazonDynamoDB rootAmazonDynamoDb,
+                                     AmazonDynamoDBStreams rootAmazonDynamoDbStreams,
                                      AWSCredentialsProvider awsCredentialsProvider,
-                                     List<MTRecord> expectedMTRecords) {
-        this.mtAmazonDynamoDB = mtAmazonDynamoDB;
-        if (rootAmazonDynamoDBStreams != null) {
-            streamWorker = new StreamWorker(rootAmazonDynamoDB,
-                rootAmazonDynamoDBStreams,
+                                     List<MtAmazonDynamoDb.MtRecord> expectedMtRecords) {
```

Comment:
why is this expanded to explicitly denote the outer class (i.e., "MtAmazonDynamoDb.MtRecord" instead of just "MtRecord")?